packages/cli/package.json

```diff
@@ -40,6 +40,18 @@
       "init": "./lib/hooks/init/inithook"
     }
   },
+  "mocha": {
+    "extension": [
+      ".test.js",
+      ".test.ts"
+    ],
+    "recursive": true,
+    "timeout": 60000,
+    "require": [
+      "../../test/hook.js",
+      "source-map-support/register"
+    ]
+  },
```

Comment:
The mocha.opts for this package should be removed like the rest?

Reply:
fixed